### ISSUE TITLE
yt-healthcheck fails MySQL 8.0.19 and later

### ISSUE THREAD

**Issue by yoku0825** [yoku0825/ytkit#40]
`yt-healthcheck` fails check_autoinc_usage and always returns CRITICAL.

```
$ bin/yt-healthcheck --version
bin/yt-healthcheck - MySQL healthcheck script - ytkit Ver. 0.2.0

$ bin/yt-healthcheck -h 172.17.0.2 -uroot
Use of uninitialized value $type in hash element at /home/yoku0825/git/ytkit/bin/../lib/Ytkit/HealthCheck.pm line 366.
Use of uninitialized value in multiplication (*) at /home/yoku0825/git/ytkit/bin/../lib/Ytkit/HealthCheck.pm line 366.
CRITICAL on d6137f569b35: table d1.t1 uses auto_increment column num 200.00%(1/0)"       (master)
```

Currently, check_autoinc_usage expects `datatype` and `(digits)` and `unsigned or not`, but 8.0.19 and later was changed not to show `(digits)`.

https://github.com/yoku0825/ytkit/blob/54783d2cbef62cbf8f760fa90af3cf21b9eca002/lib/Ytkit/HealthCheck.pm#L365

```
mysql> SELECT @@version;
+-----------+
| @@version |
+-----------+
| 8.0.18    |
+-----------+
1 row in set (0.00 sec)

mysql> SELECT table_schema AS table_schema, table_name AS table_name, column_name AS column_name, auto_increment AS auto_increment, column_type AS column_type FROM information_schema.tables JOIN information_schema.columns USING(table_schema, table_name) WHERE table_schema NOT IN ('mysql', 'information_schema', 'performance_schema', 'sys');
+--------------+------------+-------------+----------------+---------------------+
| table_schema | table_name | column_name | auto_increment | column_type         |
+--------------+------------+-------------+----------------+---------------------+
| d1           | t1         | num         |           NULL | bigint(20) unsigned |
+--------------+------------+-------------+----------------+---------------------+
1 row in set (0.01 sec)
```

```
mysql> SELECT @@version;
+-----------+
| @@version |
+-----------+
| 8.0.19    |
+-----------+
1 row in set (0.00 sec)

mysql> SELECT table_schema AS table_schema, table_name AS table_name, column_name AS column_name, auto_increment AS auto_increment, column_type AS column_type FROM information_schema.tables JOIN information_schema.columns USING(table_schema, table_name) WHERE table_schema NOT IN ('mysql', 'information_schema', 'performance_schema', 'sys');
+--------------+------------+-------------+----------------+-----------------+
| table_schema | table_name | column_name | auto_increment | column_type     |
+--------------+------------+-------------+----------------+-----------------+
| d1           | t1         | num         |           NULL | bigint unsigned |
+--------------+------------+-------------+----------------+-----------------+
1 row in set (0.01 sec)
```

**Comment by yoku0825** [comment]
Fixed by https://github.com/yoku0825/ytkit/commit/c95be1c803d45025e6c034069fe1f87329b1f874

**Comment by yoku0825** [comment]
Fixed in ytkit-0.2.0-6
